### PR TITLE
[3.9.1] Automatic title at Latest Actions Backend Module

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #22921

### Summary of Changes

Fix the automatic title option at the Latest Actions Backend Module

### Testing Instructions

Go to Extensions > Modules > Administrator > Latest Actions > Advanced > Automatic Title (ON).
Next go to System > Control panel.

### Expected result

Just an updated title: `Last %s Actions` (where %s is replaced by the count of actions to show)

### Actual result

error

### Documentation Changes Required

none

### CODE REVIEW

**Review by brianteeman — APPROVED** [review]
string is ok

**Comment by Quy** [comment]
Does it matter if the strings are different `Last 5 Actions` vs `Latest Actions`?

**Comment by zero-24** [comment]
> Does it matter if the strings are different Last 5 Actions vs Latest Actions?

I don't get the question?As is the intend to change the title ;) Or do you mean en-GB wise?

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 7d03e0a30090c87b86b1b43cfb4722d825efa9b7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22925">issues.joomla.org/tracker/joomla-cms/22925</a>.</sub>

**Comment by jsubri** [comment]
I have tested this item :white_check_mark: successfully on 7d03e0a30090c87b86b1b43cfb4722d825efa9b7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22925">issues.joomla.org/tracker/joomla-cms/22925</a>.</sub>

**Comment by infograf768** [comment]
Please add strings.
`return JText::plural('MOD_LATESTACTIONS_TITLE', $params->get('count', 5));`
when using `JText::plural` we expect multiple language strings.

Example
`return JText::plural('MOD_LOGGED_TITLE', $params->get('count', 5));`

Strings
```
MOD_LOGGED_TITLE="Last Logged-in Users"
MOD_LOGGED_TITLE_1="Last Logged-in User"
MOD_LOGGED_TITLE_MORE="Last %s Logged-in Users"
```

File: administrator/language/en-GB/en-GB.mod_latestactions.ini [hunk 1]
```diff
@@ -8,4 +8,7 @@ MOD_LATESTACTIONS_FIELD_COUNT_LABEL="Count"
 MOD_LATESTACTIONS_FIELD_COUNT_DESC="The number of items to display (default 5)."
 MOD_LATESTACTIONS_LAYOUT_DEFAULT="Default"
 MOD_LATEST_ACTIONS_NO_MATCHING_RESULTS="No Matching Results"
+MOD_LATESTACTIONS_TITLE="Last Action"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.mod_latestactions.ini]: This should be plural
`Last Actions`

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 9ae1124696294cbc74c5530ece75ef0647e62e2b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22925">issues.joomla.org/tracker/joomla-cms/22925</a>.</sub>

**Comment by infograf768** [comment]
As the last changes only concerned lang strings, no need for further tests.

RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22925">issues.joomla.org/tracker/joomla-cms/22925</a>.</sub>

**Pull request merged**